website/docs/components/alert/index.md

```diff
@@ -6,6 +6,7 @@ status: released
 links:
   figma: https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=1377%3A11987
   github: https://github.com/hashicorp/design-system/tree/main/packages/components/addon/components/hds/alert
+image: /assets/illustrations/components/alert.jpg
```

Comment:
@didoo before I get too far along does this match what you were thinking?

Next steps would be to update the frontmatter for all components and then do all these same adjustments for foundations as well.

Reply:
> maybe we could call the field preview-image

I initially tried that but got confused how to plumb something with a hyphen it through to `pageAttributes`

 > can you check what parts of the path can be omitted
 
unfortunately the full path seems to be required, but i can do a little bit more digging on that

Reply:
> I initially tried that but got confused how to plumb something with a hyphen it through to `pageAttributes`

if you mean here `image: page.pageAttributes.image`, this should work _I think_: `image: page.pageAttributes['preview-image']`

> > can you check what parts of the path can be omitted

for the time being, let's leave it as is (`/assets/illustrations/***`) we will investigate later if we find issues

Reply:
Given (most?) everything else in the Ember parts of the codebase is camelCase it seems worth being consistent the yaml definitions as well and doing something like `previewImage`. Does that work for you?